packages/client/src/components/Header.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
```

Comment:
component should have a corresponding test (at the very least a stub for one) in the `tests` folder
please reach out to me if you have question regarding e2e testing using playwright

Reply:
I think the larger team (or at least I) was unaware of the expectation to have a test for each component.
We'd discussed adding unit tests for business logic in utils/services, but it may be worth having a quick chat with all the devs to make sure we're all on the same page for where and when e2e tests should be included.

Reply:
The expectation is to have applications well tested. This in some (but maybe not all) cases should include e2e/component tests as well. This is something new and would start of slowly but want to get into the habit of at least thinking about writing them and the possible things to test there. Nav bar may seem simple but its a good sanity check for things even like auth where some menus are shown/hidden depending on the user etc.. In any case yes it is something to be discussed further with the team.

Reply:
added small reference test